test/quantization/pt2e/test_numeric_debugger.py

```diff
@@ -121,3 +123,40 @@ def test_re_export_preserve_handle(self):
         debug_handle_map = _extract_debug_handles(m_export)
 
         self.assertEqual(debug_handle_map, debug_handle_map_ref)
+
+    def test_prepare_for_propagation_comparison(self):
+        m = TestHelperModules.Conv2dThenConv1d()
+        example_inputs = m.example_inputs()
+        m = capture_pre_autograd_graph(m, example_inputs)
+        generate_numeric_debug_handle(m)
+        m_logger = prepare_for_propagation_comparison(m)
+        ref = m(*example_inputs)
+        res = m_logger(*example_inputs)
+
+        from torch.ao.quantization.pt2e.numeric_debugger import OutputLogger
+        assert any(isinstance(m, OutputLogger) for m in m_logger.modules())
```

Comment:
`any` is not quite specific, perhaps we should check `all` instead? Or maybe find a node of a particular function, and make sure that a call_module after it is an OutputLogger?

Reply:
sure, we can't check all because not all modules are output loggers, but I can check the node name of loggers contains conv2d